config/locales/views/users/en.yml

```diff
@@ -45,3 +44,5 @@ en:
       reset: "Reset my API key"
       unavailable: "Unavailable"
     verify_settings_update: 'Your settings have been updated!'
+    time_zone: "Time Zone:"
```

Comment:
This should be an attribute name (under `locales/models/users`). Look up how we use `human_attribute_name` elsewhere in the code.